pkg/envoy/proxy.go

```diff
@@ -80,6 +81,15 @@ func NewProxy(cfg ProxyConfig) Proxy {
 	}
 }
 
+func (e *envoy) IsLive() bool {
+	adminPort := uint32(e.Config.ProxyAdminPort)
+	if info, err := GetServerInfo(adminPort); err == nil && info.State == envoyAdmin.ServerInfo_LIVE {
```

Comment:
should we log the error? May make for a hard debugging problem otherwise

Reply:
done.